sky/packages/sky/lib/rendering/box.dart

```diff
@@ -1395,48 +1395,40 @@ class RenderImage extends RenderBox {
   }
 
   Size _sizeForConstraints(BoxConstraints constraints) {
+    constraints = new BoxConstraints.tightFor(
+      width: _width,
+      height: _height
+    ).apply(constraints);
+
     if (constraints.isTight)
       return constraints.constrain(Size.zero);
```

Comment:
There should be a BoxConstraints getter that returns new Size(minWidth, minHeight), or some such.